src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -293,8 +299,9 @@ function PaymentMethodList({
                     icon={item.icon}
                     disabled={item.disabled}
                     iconFill={item.iconFill}
-                    iconHeight={item.iconSize}
-                    iconWidth={item.iconSize}
+                    iconHeight={item.iconHeight || item.iconSize}
+                    iconWidth={item.iconWidth || item.iconSize}
+                    iconStyles={item.iconStyles}
```

Comment:
NAB. Shouldn't we apply same styles here

https://github.com/Expensify/App/blob/1bfd5f10ceedec1566b1a57aa08a10a8d6d5daa8/src/pages/ReimbursementAccount/EnableStep.js#L86-L87

https://github.com/Expensify/App/blob/1bfd5f10ceedec1566b1a57aa08a10a8d6d5daa8/src/components/AddPlaidBankAccount.js#L236-L237